habitat/core/env.py

```diff
@@ -91,7 +91,13 @@ def __init__(
             self._setup_episode_iterator()
             self.current_episode = next(self.episode_iterator)
             self._config.defrost()
-            self._config.SIMULATOR.SCENE = self.current_episode.scene_id
+            self._config.SIMULATOR.SCENE_DATASET = self._dataset.episodes[
```

Comment:
Why we decided to use episode 0 instead of Iterator next episode? This line will cause error when iterator isn't using 0 as first one.

Reply:
Good catch, thanks!